src/compute-model-statistics/src/main/scala/ComputeModelStatistics.scala

```diff
@@ -113,11 +109,50 @@ class ComputeModelStatistics(override val uid: String) extends Transformer with
   /** @group setParam */
   def setEvaluationMetric(value: String): this.type = set(evaluationMetric, value)
 
-  lazy val logger = Logger.getLogger(this.getClass.getName)
+  // Parameter required only is using native Spark ML Estimator instead of MMLSpark Estimator
+  val modelName: Param[String] =
+    StringParam(this, "modelName", "Model name, only required if using SparkML estimators", "SparkML")
```

Comment:
Using a String type to represent an object this is brittle, could you use use a pipeline stage param?

Reply:
I've removed the param now